src/build.py

```diff
@@ -788,23 +791,35 @@ def CMakeCommandBase():
     command.append('-DCMAKE_EXPORT_COMPILE_COMMANDS=ON')
     command.append('-DCMAKE_BUILD_TYPE=Release')
     if IsMac():
-        # Target macOS Seirra (10.12)
-        command.append('-DCMAKE_OSX_DEPLOYMENT_TARGET=10.12')
+        command.append('-DCMAKE_OSX_DEPLOYMENT_TARGET=%s' % OSX_TARGET_VERSION)
     elif IsWindows():
         # CMake's usual logic fails to find LUCI's git on Windows
         git_exe = proc.Which('git')
         command.append('-DGIT_EXECUTABLE=%s' % git_exe)
     return command
 
 
-def CMakeCommandNative(args):
+def CMakeCommandNative(args, build_dir):
     command = CMakeCommandBase()
     command.append('-DCMAKE_INSTALL_PREFIX=%s' % GetInstallDir())
     if IsLinux() and host_toolchains.ShouldUseSysroot():
         command.append('-DCMAKE_SYSROOT=%s' % GetPrebuilt(LINUX_SYSROOT))
         command.append('-DCMAKE_EXE_LINKER_FLAGS=-static-libstdc++')
         command.append('-DCMAKE_SHARED_LINKER_FLAGS=-static-libstdc++')
 
+    elif IsMac() and host_toolchains.ShouldUseSysroot():
+        # Create relpath symlink.
+        print("Creating symlink to %s" % os.path.join(build_dir, 'xcode_sdk'))
+        symlink_dest = os.path.join(build_dir, 'xcode_sdk')
+        if not os.path.exists(symlink_dest):
+            print("creating symlink %s", symlink_dest)
+            os.symlink('/Library/Developer/CommandLineTools/SDKs', symlink_dest)
```

Comment:
I wasn't sure if there was a programmatic way to get '/Library/Developer/CommandLineTools/SDKs' which is why it's hard-coded here.

If there's a better way to do this let me know.

Reply:
BTW the emscripten-releases builder also has  the path '/Library/Developer/CommandLineTools/SDKs'.

Reply:
FWIW my mac laptop does not have this path. The command line tools must be located somewhere else on my machine.

Reply:
Ideally, there should be some way to determine how Cmake determines the sysroot on Mac.

Reply:
OK, I think I have it:
`xcrun --show-sdk-path` prints the path to the current SDK. On my machine that's
'/Users/dschuff/code/osx_sdk/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.14.sdk'
So we can just run that to find the SDKs path.